lib/widgets/action_sheet.dart

```diff
@@ -34,10 +34,14 @@ import 'text.dart';
 import 'theme.dart';
 import 'topic_list.dart';
 
-void _showActionSheet(
```

Comment:
nit:

    The Figma has a "context menu" component (see link in new dartdoc),
    which _showBottomSheet has been a good fit for. (See e.g.

should this say "_showActionSheet"?